examples/docker/shiny-core/README.md

```diff
@@ -0,0 +1,58 @@
+# Model monitoring application
+
+This app is built on Shiny and allows you to visualize model training accuracy scores in realtime in 2 steps:
+
+* Train multiple models with different parameters locally, and store the accuracy scores to a remote database at the end of every epoch.
+* The visualisation application dynamically detects changes in the remote database and generates a plot as the training progresses.
+
+## Connecting to MongoDB Cloud
+
+For the purpose of this tutorial we will use the `MongoDB Atlas` cloud database. Let's see the steps for creating a cluster:
```

Comment:
mention that this service allows you to host a database for free

Reply:
[fixed](https://github.com/ploomber/doc/pull/105/files#diff-883b5ca3546bed48155f3709e49ebe5d5b987079fdc8b9c9d9f930a943512616R10-R11)